CONTRIBUTING.md

```diff
@@ -7,7 +7,12 @@ If you are interested in contributing to TorchServe, your contributions will fal
 2. You want to implement a feature or bug-fix for an outstanding issue.
     - Search for your issue here: https://github.com/pytorch/serve/issues
     - Pick an issue and comment on the task that you want to work on this feature.
-   - If you need more context on a particular issue, please ask.
+    - To ensure your changes doesn't break any of the existing features run the sanity suite as follows from serve directory:
+    ```bash
+    ./torchserve_sanity.sh
+    ```
+    - For running individual test suites refer [code_coverage](docs/code_coverage.md) documentation
+    - If you need more context on a particular issue, please ask.
```

Comment:
Can we change this to the slack channel reference and other means of communication. **please ask** is vague

Reply:
Done.